cmd/bootstrap/main.go

```diff
@@ -1,7 +1,16 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"os"
+
+	"aspect.build/cli/cmd/bootstrap/root"
+)
 
 func main() {
-	fmt.Println("Hello, World!")
+	rootCmd := root.NewRootCmd()
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Fprintf(os.Stderr, "Whoops. There was an error while executing your CLI '%s'", err)
```

Comment:
is this a "you did it wrong" or a "file a bug" message? I think the latter, this should never happen regardless of user input?

Reply:
I agree. Do we have standard language for these errors? I will update this accordingly.

Reply:
Something about it being non-user-caused and what to do about it. Maybe sth like "Internal Error, please file an issue, %v"

Looking around the root of the aspect command, I don't see a similar error message to copy from though, it just prints ERROR and exits 1

Reply:
Ok. I will add a template for this and use it.

Reply:
I'm sure our Go expert @f0rmiga could point to some idiom for this, seems like aspecterrors/errors.go could provide some InternalError or something.

Reply:
That is what I was thinking.

Reply:
To avoid blowing up this PR, I add #251. In the meantime, I added `aspecterrors.HandleError()` and updated `aspect` and `bootstrap` to use it for their top-level error handling.